src/components/Search/SearchRouter/getQueryWithSubstitutions.ts

```diff
@@ -0,0 +1,51 @@
+import type {SearchAutocompleteQueryRange} from '@components/Search/types';
+import * as parser from '@libs/SearchParser/autocompleteParser';
+
+type SubstitutionEntry = {value: string};
```

Comment:
Do we need the value key here? AFAIK we always have the map as `{[filterKey:typedValue]: {value: [id]}}`. Could we simplify it to `{[filterKey:typedValue]: [id]}`?
```suggestion
type SubstitutionEntry = {value: string};
```

Reply:
I used to think we might need more info in this map, but since the current solution is working then I will simplify